src/System.Private.CoreLib/shared/System/SpanHelpers.Char.cs

```diff
@@ -175,6 +175,206 @@ public static unsafe int IndexOf(ref char searchSpace, char value, int length)
             }
         }
 
+        public static unsafe int IndexOfAny(ref char searchSpace, char value0, char value1, int length)
```

Comment:
Do we have test coverage for corner cases this code might affect?

Reply:
via string.IndexOfAny but I have added testing via the Span entry points which should give fuller coverage https://github.com/dotnet/corefx/pull/32069